scapy/contrib/scada/iec104/__init__.py

```diff
@@ -0,0 +1,16 @@
+from scapy.contrib.scada.iec104.iec104_fields import *  # noqa F403,F401
+from scapy.contrib.scada.iec104.iec104_information_elements import *  # noqa F403,F401
+from scapy.contrib.scada.iec104.iec104_information_objects import *  # noqa F403,F401
+from scapy.contrib.scada.iec104.iec104 import *  # noqa F403,F401
+
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Thomas Tannhaeuser <hecke@naberius.de>
+# This program is published under a GPLv2 license
+#
+# scapy.contrib.description = IEC-60870-5-104 APCI / APDU layer definitions
+# scapy.contrib.status = loads
+
+
+# Package of contrib SCADA IEC-60870-5-104 specific modules
+# that have to be loaded explicitly.
```

Comment:
It might be easier if the whole module (all files) gets loaded when this does.

There’s another PR adding some scada protocols: https://github.com/secdev/scapy/pull/1898

Let’s merge this one ASAP so that the folder hierarchy is ready. I also wonder if importing `scada` should get all submodules

Reply:
> It might be easier if the whole module (all files) gets loaded when this does.

It is - the init already imports all related files. I will remove the misleading phrase "that have to be loaded explicitly"

> I also wonder if importing `scada` should get all submodules

added 

```
from scapy.contrib.scada.iec104 import *
```
on the scada-package-level so importing scada also imports iec104